stl/inc/yvals_core.h

```diff
@@ -146,7 +146,8 @@
 // P0550R2 remove_cvref
 // P0553R4 <bit> Rotating And Counting Functions
 // P0556R3 <bit> ispow2(), ceil2(), floor2(), log2p1()
-//            (log2p1() is called bit_length() as of D1956)
+//            (called has_single_bit, bit_ceil, bit_floor, and bit_width as of
+//             P1956R1 adoption)

```

Comment:
Should we include P1956R1 in this list?

Reply:
This case is special because they renamed the functions we were using as P0556's name. I guess that's what I deserve for trying to improve a reasonably clear title (some paper titles are desperately in need of improvement; this one wasn't). My suggestion is to list the papers as:

```
P0556R3 <bit> Integral Power-Of-2 Operations (renamed by P1956R1)
P1956R1 <bit> has_single_bit(), bit_ceil(), bit_floor(), bit_width()
```